pyemd/__init__.py

```diff
@@ -33,6 +33,14 @@
     >>> emd_with_flow(first_signature, second_signature, distance_matrix)
     (3.5, [[0.0, 0.0], [0.0, 1.0]])
 
+You can also calculate the EMD directly from two arrays:
```

Comment:
Could elaborate and say “arrays of observations” or “samples” instead of “arrays” and mention that the histogram is computed automatically.